spec/policies/trainee_policy_spec.rb

```diff
@@ -110,7 +110,7 @@ def user_with_organisation(user, organisation)
       it { is_expected.not_to permit(provider_user, provider_trainee) }
       it { is_expected.not_to permit(other_provider_user, provider_trainee) }
       it { is_expected.not_to permit(lead_school_user, provider_trainee) }
-      it { is_expected.not_to permit(system_admin_user, provider_trainee) }
+      it { is_expected.to permit(system_admin_user, provider_trainee) }
```

Comment:
we've changed the login so a system admin can now always withdraw a trainee regardless of their state